.circleci/install_dependencies.sh

```diff
@@ -20,6 +20,14 @@ set -e
 
 GREEN() { echo -e "\033[0;32m$1\033[0m"; }
 
+echo $(GREEN "Installing Bazel...")
+apt-get install -y apt-transport-https curl gnupg
+curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
+mv bazel.gpg /etc/apt/trusted.gpg.d/
+echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
+apt-get update
+apt-get install bazel
+
```

Comment:
This will potentially slow down all 20 parallel CI jobs instead of just the one `Validator Tests` job that needs it, so I think it should be added inline as originally suggested.

**New idea:**
- Add a new shell script similar to `install_dependencies.sh` called `install_validator_dependencies.sh`
- Do all setup (protobuf installation, bazel setup, etc.) in that script
- Replace the `Install Protobuf` step with `Install Validator Dependencies` and invoke the new shell script

Reply:
Done.